src/Backoffice/Employee/App/Requests/StoreEmployeeRequest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Backoffice\Employee\App\Request;
+
+use Illuminate\Foundation\Http\FormRequest;
+use Lightit\Backoffice\Employee\Domain\DataTransferObjects\EmployeeDto;
+
+class StoreEmployeeRequest extends FormRequest
+{
+    public const NAME = 'name';
+
+    public const EMAIL = 'email';
+
+    /**
+     * @return array<string, mixed>
+     */
+    public function rules(): array
+    {
+        return [
+            self::NAME => ['required', 'string', 'max:255'],
+            self::EMAIL => ['required', 'email:strict', 'unique:employees'],
```

Comment:
To be more Eloquent-related:

```suggestion
            self::EMAIL => ['required', 'email:strict', 'Rule::unique(Employee::class)'],
```